docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -48,8 +48,8 @@ with the https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansibl
 endif::[]
 ifeval::["{build}" != "upstream"]
 . During the overcloud deployment, RGW is applied at step 2
-(external_deployment_steps), and a cephadm compatible spec is generated in
-`/home/ceph-admin/specs/rgw` from {OpenStackPreviousInstaller}. Find the RGW spec:
+(external_deployment_steps), and a `cephadm` compatible spec is generated in
+`/home/ceph-admin/specs/rgw` from {OpenStackPreviousInstaller}. Locate the RGW spec:
```

Comment:
Do you mean step 2 of this procedure?

And what does "(external_deployment_steps)" refer to?

Reply:
mm this is a tripleo specific term, we can remove it to hide this detail for customers.